### PR TITLE
WS2-2194: Changed banners.twig, button-cta.twig, field--block-content-field-heading.html.twig and heading.twig for accessibility

### PR DESCRIPTION
**Pull request by juanmitriatti**
#### Solution 
Changed banners.twig, button-cta.twig, field--block-content-field-heading.html.twig and heading.twig for accessibility

### Links

- [JIRA ticket](https://asudev.jira.com/browse/WS2-2194)

### Checklist

- [ ] Design updates match [Web Standards](https://xd.adobe.com/view/56f6cb78-9af5-4b12-b4ce-ef319f71113f-03a5/) and [Unity Design System](https://unity.web.asu.edu)
- [ ] Solution is documented on the Jira ticket
- [ ] QA steps to verify have been included on the Jira ticket
- [ ] No new PHP or JS errors
- [ ] No accessibility issues are introduced with this update
- [ ] Added/updated README.md files, if relevant
- [ ] Confirm that any yaml files included have a matching update hook

### Verified in browsers 

- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

### Screenshots

Note: Sections that are not applicable for this issue can be removed.

### CODE REVIEW

**Review by mlsamuelson — CHANGES_REQUESTED** [review]
@juanmitriatti The heading sizing is off in the banner on https://pr-531-webspark-ci.ws.asu.edu/kitchen-sink after this update. I assume this depends on the `unity-bootstrap-theme` styles being updated. Does the release with those updates need to be included with this PR?

**Review by tbutterf — CHANGES_REQUESTED** [review]
Just one thing that I think needs to be fixed. Otherwise, looks good.

File: web/themes/webspark/renovation/templates/field/field--block-content--field-heading.html.twig [hunk 1]
```diff
@@ -14,7 +14,7 @@
 {% for item in items %}
   <div{{ attributes.addClass(classes) }}>
     {% include '@renovation/headings/heading.twig' with {
-      html_tag : 'h3',
+      html_tag : 'h1',
```
[the remark this answers is not in the document]
Comment (juanmitriatti) [reply web/themes/webspark/renovation/templates/field/field--block-content--field-heading.html.twig]: @tbutterf  this is now ready. 
Thanks for your feedback.

**Review by tbutterf — APPROVED** [review]

**Comment by juanmitriatti** [comment]
> need to be included with this PR?

Hi @mlsamuelson 
It depends on the` unity-bootstrap-theme`. This is the PR associated on the UDS side : https://github.com/ASU/asu-unity-stack/pull/1330/files . It needs to be included.
Note : Yesterday. I did the upgrade from the unity boostrap theme.
Thanks

**Review by mlsamuelson — APPROVED** [review]

**Pull request merged**